bff/src/http/routes/index.ts

```diff
@@ -16,16 +16,16 @@ const httpBasicProtectedRouter = createRouter();
 
 unprotectedRouter
   .get('/', (_req, res) => res.send('Hello World!'))
-  .use(sessionRouter);
+  .use(sessionRouter)
+  .use('/google/v1', googleRouter);
```

Comment:
We should move it to the protected router right?

Reply:
Yes.. sorry :(